connection.js

```diff
@@ -1573,9 +1573,28 @@ Connection.prototype.queue_msg = function (retval, msg) {
     }
 };
 
+Connection.prototype.store_queue_result = function (retval, msg) {
+    var plugin = {name: 'queue'};
+    switch (retval) {
+        case constants.ok:
+            this.transaction.results.add(plugin, { pass: msg });
+            break;
+        case constants.deny:
+        case constants.denydisconnect:
+        case constants.denysoft:
+        case constants.denysoftdisconnect:
+            this.transaction.results.add(plugin, { fail: msg });
+            break;
+        default:
+            this.transaction.results.add(plugin, { msg: msg });
```

Comment:
With ```CONT``` returned (Haraka handles outbound), ```queue``` results object will be logged with msg set to null.